news/2025/2025-02-27-introducing-teams.md

```diff
@@ -0,0 +1,74 @@
+
+## What's new
+
+The [last osu!(lazer) update](https://osu.ppy.sh/home/news/2025-02-20-osulazer-updates-february-20-2025) introduced a new "Teams" feature. This is a new way for the community to express shared interests and proudly display groups they are part of.
+
+![](/wiki/shared/news/2025-02-27-introducing-teams/team-banner.jpg)
+
+Teams are shown beside usernames in places like user profiles, leaderboards, and forums. Each team gets a flag and name to display everywhere, a unique acronym to go along with it, a banner and description, and their own internal leaderboard.
+
+![](/wiki/shared/news/2025-02-27-introducing-teams/team-display.png)
+
+Users can interact with all of these features by going to the ["My Team"](https://osu.ppy.sh/teams/create) page at any time. Players not yet in a team will be given the option to create one. Alternatively, by going to the page of an existing team, users will be able to apply to any team they see fit!
+
+### Current features
+
+While the feature is still in its infancy, there are plenty of things users can already do with teams:
+
+- Create and join other users' teams.
+- Manage incoming applications.
+- Compete with your peers in the team's leaderboard.
+- Communicate with your team in a private chatroom.
+- Proudly display your allegiance with a team flag and unique acronym!
+
+### Upcoming features
+
+You can expect to see more features in the future as the system is refined. While there may be more to come, these are a few of the things to look forward to:
+
+- A listing on the website with all available teams in the platform.
+- Displaying team leaderboards in osu!(lazer).
+- [osu!api (v2)](https://osu.ppy.sh/docs/index.html) support for teams.
+
+### Limitations
+
+- Team acronyms are unique.
+- Both team acronyms and names are permanent (contact [support](mailto:accounts@ppy.sh) or re-create your team if you make a mistake!).
+- Team leaders cannot be swapped.
+- The maximum team size is dependent on the amount of [osu!supporters](/wiki/osu!supporter) on the team.
+  - Each supporter will add 4 slots to the base team size of 8, up to a cap of 256.
+- Users can only join one team at a time.
+  - To join another team, you may do so at any time by leaving your current one.
+
+Some of these limitations may be lifted as the system is iterated upon, a lot of which may depend on the community's feedback. Don't be afraid to propose and discuss changes in the [osu!web GitHub repository](https://github.com/ppy/osu-web/issues?q=is:issue%20state:open%20teams%20label:area:team)!
+
+In the meantime, if team leaders wish to change their team's name, acronym, or even the leader itself, they may do so by contacting the [account support team](/wiki/People/Account_support_team) via [accounts@ppy.sh](mailto:accounts@ppy.sh).
+
+## Stats
+
+It's hard to put into perspective how much the community has grown to love teams ever since their release despite their recency. So why not put it into numbers instead!
```

Comment:
```suggestion
It's hard to put into perspective how much the community has grown to love teams ever since their release, despite their recency. Let's see if numbers will help with that!
```